rcl/test/rcl/test_arguments.cpp

```diff
@@ -392,7 +437,7 @@ TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_param_argument_
 
 TEST_F(CLASSNAME(TestArgumentsFixture, RMW_IMPLEMENTATION), test_param_argument_multiple) {
   const char * argv[] = {
-    "process_name", "__params:=parameter_filepath1", "__ns:=/namespace",
+    "process_name", "--ros-args", "__params:=parameter_filepath1", "__ns:=/namespace",
```

Comment:
I see a lot of tests being updated, but are there new tests that test cases like "no trailing `--`" versus "having a trailing `--`" or "multiple pairs of `--ros-args`/`--`" or even something like `exec arg --ros-args foo:=bar -- another --ros-args __ns:=my_ns` (two, one with and without `--`)?

Maybe I missed where those are or maybe they were overlooked. I think it would be good to have test cases for those kinds of things.

Reply:
Yes! Those were in the queue, along with the patches for downstream packages. Just pushed them, see 8738f9b27c78aab3458df55b3d448aca745b9eae.